accelerators/sunbirdrc/src/main/java/org/egov/sunbirdrc/service/AddVcSchemaService.java

```diff
@@ -0,0 +1,119 @@
+        ResponseEntity<String> responseEntity = new RestTemplate().exchange(SCHEMA_ENDPOINT, HttpMethod.POST, requestEntity, String.class);
+        String addSchemaResponse=responseEntity.getBody();
+        String schemaId=getSchemaIdFromResponse(addSchemaResponse);
+        // Print the response received from the request
+        System.out.println("Response from schema endpoint and schema id : " + responseEntity.getBody());
+        System.out.println("schema id : " + schemaId);
+
+        System.out.println(addVcSchemaToMdms(credentialSchemaPayload,schemaId));
+        return responseEntity.getBody();
+    }
+
+
+    public String getSchemaIdFromResponse(String addSchemaResponse) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        String schemaId;
+        try {
+            JsonNode responseBody = objectMapper.readTree(addSchemaResponse);
+            schemaId = responseBody.path("schema").path("id").asText();
+        } catch (IOException e) {
+            // Handle parsing errors by providing a more informative error message
+            throw new RuntimeException("no Id in the response   " + e.getMessage());
+        }
+        return schemaId;
+    }
+
+    public String addVcSchemaToMdms(JsonNode mdmsDataRequestPayload,String schemaId) throws JsonProcessingException {
+            // Initialize ObjectMapper
+            ObjectMapper objectMapper = new ObjectMapper();
+            RestTemplate restTemplate = new RestTemplate();
+            // Create the payload
+            String uuid = mdmsDataRequestPayload.path("mdmsData").path("uuid").asText();
+            JsonNode path = mdmsDataRequestPayload.path("mdmsData").path("path");
+            String did = mdmsDataRequestPayload.path("schema").path("author").asText();
+            String mdmsCodeName=mdmsDataRequestPayload.path("mdmsData").path("code").asText();
+
+            String mdmsSchemaPayload = "{\"RequestInfo\":{\"apiId\":\"Rainmaker\",\"authToken\":\"8c68a385-196a-4790-8aee-42323faef9ad\",\"userInfo\":{\"id\":595,\"uuid\":\"1fda5623-448a-4a59-ad17-657986742d67\",\"userName\":\"UNIFIED_DEV_USERR\",\"name\":\"Unified dev user\",\"mobileNumber\":\"8788788851\",\"emailId\":\"\",\"locale\":null,\"type\":\"EMPLOYEE\",\"roles\":[{\"name\":\"Localisation admin\",\"code\":\"LOC_ADMIN\",\"tenantId\":\"pg\"},{\"name\":\"Employee\",\"code\":\"EMPLOYEE\",\"tenantId\":\"pg\"},{\"name\":\"MDMS Admin\",\"code\":\"MDMS_ADMIN\",\"tenantId\":\"pg\"},{\"name\":\"SUPER USER\",\"code\":\"SUPERUSER\",\"tenantId\":\"pg\"}],\"active\":true,\"tenantId\":\"pg\",\"permanentCity\":null},\"msgId\":\"1695889012604|en_IN\",\"plainAccessRequest\":{}},\"SchemaDefinition\":{\"tenantId\":\"default\",\"code\":\"VerifiableCredentials.test2\",\"description\":\"Trade type for trade license applications\",\"definition\":{\"$schema\":\"http://json-schema.org/draft-07/schema#\",\"type\":\"object\",\"required\":[\"path\"],\"x-unique\":[\"path\"]},\"isActive\":true}}";
```

Comment:
Don't hard code request. Create the request using mdms pojos imported from mdms-client library